Gemfile

```diff
@@ -11,6 +11,9 @@ gemspec
 # Git. Remember to move these dependencies to your gemspec before releasing
 # your gem to rubygems.org.
 
+# TODO: Move this into the Gemspec
+gem "literal", path: "../literal"
```

Comment:
I will cut a release on RubyGems before this PR is ready to merge and move this up to the Gemspec.